Source/FuseJS/Tests/Environment.Test.uno

```diff
@@ -2,31 +2,32 @@ using Uno;
 using Uno.IO;
 using Uno.Testing;
 using Fuse.Scripting;
+using Fuse.Scripting.JavaScript.Test;
 using FuseTest;
 
 public class JsTest : TestBase
 {
 	[Test]
 	public void Environment()
 	{
-		new FuseJS.Environment();
-		using (var jsRecorder = new JsTestRecorder())
-		{
-			var context = jsRecorder.Begin();
-			var iOS = defined(iOS) ? "true" : "false";
-			var android = defined(android) ? "true" : "false";
-			var mobile = defined(mobile) ? "true" : "false";
-			var designmode = defined(designmode) ? "true" : "false";
+		var test = new JSTest(EnvironmentInner);
```

Comment:
This is confusing. The new helper is called `JSTest`, and this class is called `JsTest`. Notice the lower-case `s` in the latter. I think we should fix this up by renaming this class to `EnvironmentTest` instead, to match the normal filename/test-name convention here.

Reply:
agreed, on it

Reply:
done